.github/workflows/kind-e2e.yaml

```diff
@@ -93,6 +93,10 @@ jobs:
         sudo swapoff -a
         sudo rm -f /swapfile
 
+        # Use tmpfs for etcd storage
```

Comment:
Should we add the same comments as in net-contour regarding reasoning and source?

Reply:
I have added the same.